cli/azd/resources/scaffold/templates/resources.bicept

```diff
@@ -389,28 +389,23 @@ module {{bicepName .Name}} 'br/public:avm/res/app/container-app:0.8.0' = {
 {{- end}}
 
 {{- if .DbRedis}}
-module redis 'br/public:avm/res/cache/redis:0.3.2' = {
+module redis 'br/public:avm/res/cache/redis:0.9.0' = {
```

Comment:
Do we have a process yet for when/how we upgrade to new versions of AVM modules?

Reply:
That's a very good question. Do you have any suggestions? And if there's a need for something solid to be in-place today?

In general, it feels not too different to me than when we update `api-version` fħr any Azure resources, which boils down to when a version upgrade is required for a specific feature. Currently, this is mostly hidden away from the user so it wouldn't matter too much.

Reply:
I'm not sure about this, @weikanglim do you know?

Reply:
Since this is out of scope of this PR I created #4728 to continue this discussion further